docs/ops/config.rst

```diff
@@ -255,6 +255,16 @@ in other Django projects.
    If :envvar:`DJANGO_USE_OIDC` is set to ``True``, this settings must be set to
    the URL that a user can visit to logout. It may be a relative URL.
 
+.. envvar:: DJANGO_PEER_APPROVAL_ENFORCED
```

Comment:
Yeah I don't love this name, but I also don't have any useful suggestions for changing it, so r+ to this name.